migrations/versions/dceb6cd3c41e_.py

```diff
@@ -0,0 +1,54 @@
+"""Add policycondition table, remove unused condition column from policy table
+
+Revision ID: dceb6cd3c41e
+Revises: b9131d0686eb
+Create Date: 2019-07-02 12:19:19.646528
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = 'dceb6cd3c41e'
+down_revision = 'b9131d0686eb'
+
+from alembic import op, context
+import sqlalchemy as sa
+from sqlalchemy.schema import Sequence, CreateSequence
+
+
+def dialect_supports_sequences():
+    migration_context = context.get_context()
+    return migration_context.dialect.supports_sequences
+
+
+def create_seq(seq):
+    if dialect_supports_sequences():
+        op.execute(CreateSequence(seq))
+
+
+def upgrade():
+    try:
+        try:
+            create_seq(Sequence('policycondition_seq'))
+        except Exception as _e:
+            pass
+        op.create_table('policycondition',
+        sa.Column('id', sa.Integer(), nullable=False),
+        sa.Column('policy_id', sa.Integer(), nullable=False),
+        sa.Column('section', sa.Unicode(length=255), nullable=False),
+        sa.Column('key', sa.Unicode(length=255), nullable=False),
```

Comment:
upper case...

Reply:
done in 07fd2e706a9c321f1bf70066f63664afb46b8e0c